versioned_docs/version-8.3/self-managed/zeebe-deployment/operations/resource-planning.md

```diff
@@ -148,3 +156,24 @@ You should assign idempotent ids to events in your exporter if this is an issue
 ### Effect of quorum loss
 
 If a partition goes under quorum (for example, if two nodes in a 3-node cluster go down), the leader of the partition continues to accept requests, but these requests are not replicated and are not marked as committed. In this case, they cannot be truncated. This causes the event log to grow. The amount of disk space needed to continue operating in this scenario is a function of the broker throughput and the amount of time to quorum being restored. You should ensure your nodes have sufficient disk space to handle this failure mode.
+
+## Memory
+
+Memory usage is based on the Java heap size (by default [25% of the max RAM](TODO: Insert documentation here)) and native memory usage (also by default [25% of the max RAM](TODO: Insert documentation here)), so Java itself will use **up to** 50% of the maximum RAM.
+
+RocksDB will then allocate [512MB per partition](TODO: where does this value come from?) by default.
```

Comment:
The default value comes from the configuration file. We picked it based on the cluster plans that existed at the time; no idea if it still fits.

Reply:
Could you please point me to the configuration and the according rocksdb docs? I should be able to continue from there.

Reply:
@npepinpe quick reminder

Reply:
as of now it’s still in the [experimental config section](https://docs.camunda.io/docs/next/self-managed/zeebe-deployment/configuration/#experimental-configuration-options) for which you only find docs in the yaml template, the mem config is [here](https://github.com/camunda/zeebe/blob/main/dist/src/main/config/broker.yaml.template#L963).

Reply:
Thank you @megglos I am including a permalink to the file 👍